Workbooks/Virtual Machines - Performance Analysis/Performance Analysis for a Group of VMs/Performance Analysis for a Group of VMs.workbook

```diff
@@ -163,7 +163,7 @@
             "name": "Counter",
```

Comment:
Since the file name is changed in this PR, if this gets checked in before InfraInsights code change. It will break in PROD. This is exactly the reason why I had not yet modified the file name.

You have 2 options:
1. Do not change file in this PR. Complete rest of the changes in this PR. Make the file name change in a separate PR so that the down time is minimized.
2. Wait till this change goes to PROD and then complete the PR.

In both options we have try to minimize down time of this workbook.

Reply:
The Infrainsights code has been updated to handle this, it is currently in PR